### PR TITLE
Fix verify stage when specify path

### LINKED ISSUE #159: Bug theme when specify path verify stage
Hi @seebeen, thanks for the immediate response.
I did a test but with this configuration the file verification fails.

```
[
    "@semantic-release/wordpress",
    {
        "type": "theme",
        "slug" : "my-theme",
        "withReadme": true,
        "copyFiles": false,
        "releasePath": "/tmp/wp-release",
        "path": "./dist/my-theme"
    }
],
```

i think because inside `verify-theme.ts` concat path with slug.

https://github.com/semantic-release/wordpress/blob/2e63d93bdd87e8f58f5d1479bedc8b6a533dcfe9/lib/utils/verify-theme.ts#L9

So the path in verify state is "./dist/my-theme/my-theme" and in prepare state is "./dist/my-theme"

In my case the correct path is "./dist/my-theme".

```
[5:13:15 PM] [semantic-release] › ✘  ETHEMEFILENOTFOUND Your theme must contain these files: style.css, functions.php
Check if the files exist, and if not create them: style.css, functions.php.
AggregateError: 
    SemanticReleaseError: Your theme must contain these files: style.css, functions.php
        at getError (file:///builds/my-theme/theme/node_modules/@semantic-release/wordpress/dist/utils/get-error.js:5:12)
        at verifyTheme (file:///builds/my-theme/theme/node_modules/@semantic-release/wordpress/dist/utils/verify-theme.js:13:13)
        at async verifyConditions (file:///builds/my-theme/theme/node_modules/@semantic-release/wordpress/dist/verify-conditions.js:12:17)
        at async validator (file:///builds/my-theme/theme/node_modules/semantic-release/lib/plugins/normalize.js:36:24)
        at async file:///builds/my-theme/theme/node_modules/semantic-release/lib/plugins/pipeline.js:38:36
        at async Promise.all (index 0)
        at async next (file:///builds/my-theme/theme/node_modules/semantic-release/node_modules/p-reduce/index.js:15:44)
    at file:///builds/my-theme/theme/node_modules/semantic-release/lib/plugins/pipeline.js:55:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async pluginsConfigAccumulator.<computed> [as verifyConditions] (file:///builds/my-theme/theme/node_modules/semantic-release/lib/plugins/index.js:87:11)
    at async run (file:///builds/my-theme/theme/node_modules/semantic-release/index.js:106:3)
    at async Module.default (file:///builds/my-theme/theme/node_modules/semantic-release/index.js:275:22)
    at async default (file:///builds/my-theme/theme/node_modules/semantic-release/cli.js:55:5) {
  errors: [
    SemanticReleaseError: Your theme must contain these files: style.css, functions.php
        at getError (file:///builds/my-theme/theme/node_modules/@semantic-release/wordpress/dist/utils/get-error.js:5:12)
        at verifyTheme (file:///builds/my-theme/theme/node_modules/@semantic-release/wordpress/dist/utils/verify-theme.js:13:13)
        at async verifyConditions (file:///builds/my-theme/theme/node_modules/@semantic-release/wordpress/dist/verify-conditions.js:12:17)
        at async validator (file:///builds/my-theme/theme/node_modules/semantic-release/lib/plugins/normalize.js:36:24)
        at async file:///builds/my-theme/theme/node_modules/semantic-release/lib/plugins/pipeline.js:38:36
        at async Promise.all (index 0)
        at async next (file:///builds/my-theme/theme/node_modules/semantic-release/node_modules/p-reduce/index.js:15:44) {
      code: 'ETHEMEFILENOTFOUND',
      details: 'Check if the files exist, and if not create them: style.css, functions.php.',
      semanticRelease: true,
      pluginName: '@semantic-release/wordpress'
    }
  ]
```

For test this user case, the prepare.specs.ts and verify-theme.spec.ts must have the same options

```
{
      type: 'theme',
      slug: 'complete-theme',
      path: './test/fixtures/complete-theme',
      copyFiles: false,
}
```

_Originally posted by @AntoninoBonanno in https://github.com/semantic-release/wordpress/issues/120#issuecomment-1785702728_

### PR DESCRIPTION
**Pull request by AntoninoBonanno**
I fixed the verify stage (both for the theme and for the plugin) in case the path is specified from the configuration, Resolve #159 .

I also added "path" in the README.md as I believe it is a useful feature when your code is in a subdirectory or is generated by Webpack as in my case

Basically I standardized the use of the "path" configuration in the "verify", "prepare", "publish" stages.

Before:

https://github.com/semantic-release/wordpress/blob/e8b42e55b94b8431fcc90048c2cad167357511f3/test/1-verify-theme.spec.ts#L90-L97

https://github.com/semantic-release/wordpress/blob/e8b42e55b94b8431fcc90048c2cad167357511f3/test/2-prepare-theme.spec.ts#L95-L109

https://github.com/semantic-release/wordpress/blob/e8b42e55b94b8431fcc90048c2cad167357511f3/test/3-publish-plugin.spec.ts#L12-L22

After:

https://github.com/semantic-release/wordpress/blob/5cfb7e8ad3459df84ad861883e7f721951440688/test/1-verify-theme.spec.ts#L89-L97

### CODE REVIEW

File: package.json
Comment (seebeen) [inline package.json:1]: Did you update the dependencies manually?

**Review by seebeen — CHANGES_REQUESTED** [review]
Let's sort out the dependency bumps in the commit - and we can merge then.

Awseome work, btw. Much appreciated.

File: package.json
Comment (AntoninoBonanno) [reply package.json:1]: i used [npm-check-updates](https://www.npmjs.com/package/npm-check-updates)

Comment (seebeen) [reply package.json:1]: Can you remove them from the commit please, and we can merge

**Review by seebeen — APPROVED** [review]
All good  💪

**Comment by seebeen** [comment]
Hey, I just saw that none of the commits are GPG signed. Would it be too much of a hassle to [setup GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) on your end.

After that, you can use [this guide](https://webdevstudios.com/2020/05/26/retroactively-sign-git-commits/) To retroactively sign your commits in this PR.

Reason the GPG signing is enforced in this repo is to prevent potential supply chain attacks.  
WordPress powers >45% of all the websites in the world so the poential impact of supply chain poisoning is immense.

**Comment by AntoninoBonanno** [comment]
ready

**Pull request merged**